### PR TITLE
Add test+release actions and dependabot

### PR DESCRIPTION
**Pull request by bcomnes**
- Gets test to run green
- Adds a test action. Simple.
  - Includes auto merging for GitHub Action updates.
- Adds a release action.  A secret called NPM_TOKEN must be set.
  -  Lerna only supports READ + WRITE tokens.  I suggest an npm machine user is created for this so nobody is Locke into disabling OTP.

Some other minor modifications as noted in comments below.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Basic dependabot.yml file with
+# minimum configuration for two package managers
+
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: "npm"
+    directory: "/"
```
Comment (bcomnes) [inline .github/dependabot.yml:8]: This covers all packages

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - run: npm test # block publishing on
+    - run: git reset --hard && git clean -fd
+
+    # configure git
+    - run: git config --global user.email "${{ env.git_email }}"
+    - run: git config --global user.name "${{ github.actor }}"
+
+    # publish to github and npm
+    - run: npm i lerna -g
+    - run: echo //registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }} > .npmrc
```
Comment (bcomnes) [inline .github/workflows/release.yml:42]: Make sure the NPM_TOKEN secret is set to a READ + WRITE npm token (not automation token sadly).

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+on:
+  workflow_dispatch:
+    inputs:
+      newversion:
+        description: 'lerna publish {major,minor,patch}'
+        required: true
+
+env:
+  node_version: 14
+  git_email: jonathan@infinitescroll.org
```
Comment (bcomnes) [inline .github/workflows/release.yml:13]: Feel free to change this to a team IS email or something.

File: package.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "name": "@glif/modules",
   "private": false,
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --no-ci",
```
Comment (bcomnes) [inline package.json:5]: We might test turning off --no-ci and test out --hoist.

File: packages/filecoin-message/src/index.ts [hunk 5]
```diff
@@ -24,6 +24,7 @@ export interface LotusMessage {
   GasPremium: string
   GasLimit: number
   GasFeeCap: string
+  GasPrice?: string
```
Comment (bcomnes) [inline packages/filecoin-message/src/index.ts:27]: I had to add this to get the tests to pass. Okay?

File: packages/filecoin-number/package.json [hunk 6]
```diff
@@ -14,7 +14,8 @@
     "build-types": "../../node_modules/.bin/tsc --emitDeclarationOnly",
     "build": "npm run clean && npm run build-types && npm run build-node && npm run build-browser && npm run build-module",
     "prepublishOnly": "npm run build",
-    "test": "npm run build && jest",
+    "test-skip": "npm run build && ../../node_modules/.bin/jest",
```
Comment (bcomnes) [inline packages/filecoin-number/package.json:17]: The README suggests a `APIADDR` var needs to be set?  We could set this as a secret and pass this in as variable.  Please advise on how to proceed here.

File: package.json (hunk 4 above)
Comment (Schwartz10) [reply package.json:5]: Yeah ,i think as of now you have to use `--hoist` (although not sure what --no-ci does)? Looking at some of the other packages, the commands like test and build point directly to the node_modules at the repo root. If those don't end up getting hoisted, I'm pretty sure those scripts will break. For example https://github.com/glifio/modules/pull/22/files#diff-2b01dbf04a098f3ee9dcfa5163056e971f37eedb9ef53b0d838965516347a222R17

File: packages/filecoin-message/src/index.ts (hunk 5 above)
Comment (Schwartz10) [reply packages/filecoin-message/src/index.ts:27]: Yeah for now it's fine. I'll look into it later https://github.com/glifio/modules/issues/23

File: packages/filecoin-number/package.json (hunk 6 above)
Comment (Schwartz10) [reply packages/filecoin-number/package.json:17]: Gah, that's stale. We can get rid of that. 

However, `filecoin-wallet-provider` could benefit form having a secret passed as an env var (just for testing). If it's easy, could you add the secret `apiAddress` and `token` (used [here](https://github.com/glifio/modules/pull/22/files#diff-2b01dbf04a098f3ee9dcfa5163056e971f37eedb9ef53b0d838965516347a222R17))

File: .github/workflows/release.yml (hunk 2 above)
Comment (bcomnes) [reply .github/workflows/release.yml:42]: This is set

File: package.json (hunk 4 above)
Comment (bcomnes) [reply package.json:5]: I will tweak once everything is working.

File: packages/filecoin-number/package.json (hunk 6 above)
Comment (bcomnes) [reply packages/filecoin-number/package.json:17]: Will follow up on this part.

**Pull request merged**